### ISSUE TITLE
[FEATURE] Buff Indicator

### ISSUE THREAD

**Issue by marco-paulucci** [lambdaclass/curse_of_myrra#626]
## 🚀 Feature

Add an animation or effect to skill buttons that have an "active" state (buffs), and make it easy to see how long your buff will last for. 

Following #617, this could essentially be a duplicate of that functionality, but on another layer. Like this:

![image](https://github.com/lambdaclass/curse_of_myrra/assets/20408709/d7886917-d23b-4283-a931-c110e12dac08)

Note that it's only an example. Implement the visual style as you consider fit.

**Comment by marco-paulucci** [comment]
Updated graphic

**Comment by saugon** [comment]
We need to track _skill duration_ in the backend before we can do this.
